docs/networks/node-ops/access-onchain-data/access-nodes/access-node-configuration-options.md

```diff
@@ -0,0 +1,241 @@
+# Access node operator documentation
+
+# Overview
+
+This page serves as a resource for the different configuration options for Access nodes on the Flow network.
+
+<aside>
+⚠️ Nodes MUST be running `v0.32.10+` or newer to enable execution data indexing.
+
+</aside>
+
+# Setup node’s directory
+
+The node requires the following directory structure:
+
+```bash
+$ tree flow_access
+	flow_access/
+	├── bootstrap
+	│	├── private-root-information (with corresponding AN data)
+	│	└── execution-state
+	│	└── public-root-information
+	│		├── node-id
+	│		└── node-info.pub.NODE_ID.json
+	│		└── root-protocol-state-snapshot.json (the genesis data)
+	└── data (directory used by the node to store block data)
+	│	└── execution-data
+	│	└── execution-state
+
+```
+
+# Setup Data Indexing
+
+First, your node needs to download and index the execution data. There are 3 steps:
+
+1. Enable Execution Data Sync
+2. Download the root checkpoint file
+3. Configure the node to run the indexer
```

Comment:
The last missing step I forgot wasn't included:
`4. Use the indexed data in the Access API`. Let's add that in a follow up PR